iothub/device/tests/PlugAndPlay/PnpHelperTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FluentAssertions;
+using Microsoft.Azure.Devices.Client.PlugAndPlay;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Devices.Client.Tests.PlugAndPlay
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class PnpHelperTests
+    {
+        [TestMethod]
+        public void CreateRootPropertyPatch()
+        {
+            // Format:
+            //  {
+            //      "samplePropertyName": 20
+            //  }
+
+            string propertyName = "someName";
+            int propertyValue = 10;
```

Comment:
If you declare this as a string, you won't need the extra serialization in line 30. Same with other tests.

Reply:
I don't see an issue in using "int" here and serializing it below, do you?

Reply:
Line 35 checks it against the value, so we're making sure the type is maintained round-trip. I say leave it as it is.